index.js

```diff
@@ -30,6 +30,35 @@ express()
       res.send("Error " + err);
     }
   })
+  .get('/addProduct', (req, res) => res.render('pages/addProduct'))
+  .post('/addProduct', jsonParser, async function(req, res) {
+    try {
+      const client = await pool.connect();
+      client.query(`insert into fridge_products
```

Comment:
When we insert into the fridge, would we need to find the ID and add it as well?

Reply:
User_ID is hardcoded as 1 for all queries in this sprint since there is no user login set up.  
prod_ID is being found from the products table based on the product name the user inputs.  This is then put into fridge_products.